examples/angular2/services/store.ts

```diff
@@ -1,9 +1,5 @@
-/// <reference path="../typings/node-uuid/node-uuid.d.ts" />
-/// <reference path="../typings/storejs/storejs.d.ts" />
-/// <reference path="../typings/angular2/angular2.d.ts" />
-
-import {Injectable} from 'angular2/angular2';
 import * as uuid from 'node-uuid';
+import 'store.js';
```

Comment:
> store.js exposes a commonjs interface, so why aren't we using that?

My understanding is that the type definitions for store.js define a global variable rather than a module:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/storejs/storejs.d.ts#L21

The import statement `import 'store.js';` is transpiled into a commonjs `require` via the typescript compiler.

To be honest, I'm not sure why this example uses `store.js` rather than `localStorage` directly? However, I was working on the principle of making the minimum changes possible for an Angular 2 beta upgrade.